### ISSUE TITLE
S and M size for 4x upscalers?

### ISSUE THREAD

**Issue by arianaa30** [bloc97/Anime4K#215]
I see there are various sizes available for 2x CNN upscalers. But do we have S and M sizes for 4x and 3x upscalers as well? I didn't notice anything.

**Comment by Tama47** [comment]
No, explained here https://github.com/bloc97/Anime4K/issues/213#issuecomment-1817352320
